code/modules/antagonists/blob/overmind.dm

```diff
@@ -63,6 +63,7 @@ GLOBAL_LIST_EMPTY(blob_nodes)
 		blob_core.update_icon()
 	SSshuttle.registerHostileEnvironment(src)
 	. = ..()
+	ADD_TRAIT(src, TRAIT_SIXTHSENSE, INNATE_TRAIT)
 	START_PROCESSING(SSobj, src)
 
```

Comment:
maybe add a short deadchat_broadcast that mentions to ghosts that the blob can communicate with them?

Reply:
we didnt for revenant, theyll figure it out

Reply:
add one for revenants too, then

Reply:
people already figured it out, i don't think it's necessary